include/libfreenect2/frame_listener.hpp

```diff
@@ -54,11 +54,17 @@ class LIBFREENECT2_API Frame
   size_t height;          ///< Number of lines in the frame.
   size_t bytes_per_pixel; ///< Number of bytes in a pixel.
   unsigned char* data;    ///< Data of the frame (aligned).
+  float exposure;         ///< Get the exposure time set by the color camera.
+  float gain;             ///< Get the gain set by the color camera.
+  float gamma;            ///< Get the gamma level set by the color camera.
 
   Frame(size_t width, size_t height, size_t bytes_per_pixel) :
     width(width),
     height(height),
-    bytes_per_pixel(bytes_per_pixel)
+    bytes_per_pixel(bytes_per_pixel),
+    exposure(30.f),
```

Comment:
Yes, something along those lines. 66.47 in milliseconds would be 15 Hz, and I remember something that the color camera does indeed drop the framerate to 15 Hz in low-light situations. So default at 30 Hz would then be ~ 33.3.